notebooks/hello-npu/hello-npu.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>which one googlenet-v4? Could you please provide link?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
I see, I just picked up this information from [OpenVINO documentation](https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/npu-device.html#limitations). I think I will remove this line as it doesn't say much and I am using resnet50 anyways for classification tasks.